Tests/Functions/Private/Update-PSKoanFile.Tests.ps1

```diff
@@ -0,0 +1,74 @@
+#Requires -Modules PSKoans
+
+InModuleScope 'PSKoans' {
+    Describe 'Update-PSKoanFile' {
+        BeforeAll {
+            Mock Get-PSKoanFilePath {
+                [PSCustomObject]@{
+                    UserFilePath   = Join-Path $TestDrive 'Koans\AboutSomething.ps1'
+                    ModuleFilePath = Join-Path $TestDrive 'Module\AboutSomething.ps1'
+                }
+            }
+            New-Item 'TestDrive:\Koans' -ItemType Directory

```

Comment:
Probably best to just use `TestDrive:Koans` or else make use of `[IO.Path]::DirectorySeparatorChar`